### PR TITLE
[visq] Add directory for unittest

### PR DESCRIPTION
**Pull request by jinevening**
This adds unittests for visq.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/9694
Draft PR: https://github.com/Samsung/ONE/pull/9868

### CODE REVIEW

File: compiler/visq/test/testPalette.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Test visqlib.Palette module'''
+
+import unittest
+
+from visqlib.Palette import YLORRD9Palette
```
Comment (seanshpark) [inline compiler/visq/test/testPalette.py:18]: how is `YLORRD9Palette` installed?

Comment (jinevening) [reply compiler/visq/test/testPalette.py:18]: It's in the `visq/visqlib/Palette.py`

https://github.com/Samsung/ONE/blob/master/compiler/visq/visqlib/Palette.py

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**